### DISCUSSION TITLE
Is this project still alive?

### DISCUSSION THREAD

**Discussion by paletochen** [Phambili-Tech/Newt_Display#65] [General]
Can we expect bugs fixed and/or new feature requests?
I am still enjoying my 2 devices though they could be polished a bit more.

Thanks

**Comment by chaseadam** [comment]
Are there specific features/polish you were hoping for?

**Comment by darianbjohnson** [comment]
The project is definitely still active (I just upgraded the base code to the latest esp arduino code base) and I’m slowly working on bug fixes. I’d love to hear about any features/polish that should be prioritized.

**Comment by paletochen** [comment]
Thanks for your quick answer.

I am biased so I guess I'd love to have things that I proposed accepted ;)

- [Allow to set Monday as start of the Week](https://github.com/Phambili-Tech/Newt_Display/issues/62)
- [vscode and/or esptool flashing](https://github.com/Phambili-Tech/Newt_Display/discussions/60)

[Not reported]
I have one device on the oven and often i don't check the battery, so i find the device either working or dead, which then reminds me to charge it. It would be great if, before dying, the Newt would maybe disable all features but the clock and change the screen, which would indicate that it is in its last moments. Maybe this disabling would also allow the device to last a couple days more.
Or in other words, some sort of warning as the battery indicator in its last step lasts several days 

Then I think they are also very valuable things like:

- [Different sounds for buzzer/alarms](https://github.com/Phambili-Tech/Newt_Display/issues/33)

There are also other feature requests in the Issues section

**Comment by paletochen** [comment]
10 months have passed since this thread. Can I already assume this project will not receive any updates and move on?
